pd/src/main.rs

```diff
@@ -306,16 +306,22 @@ async fn main() -> anyhow::Result<()> {
                                     name: v.name.clone(),
                                     website: v.website.clone(),
                                     description: v.description.clone(),
-                                    funding_streams: v
-                                        .funding_streams
-                                        .iter()
-                                        .map(|fs| FundingStream {
+                                    funding_streams: FundingStreams::try_from(
+                                        v.funding_streams
+                                            .iter()
+                                            .map(|fs| {
+                                                FundingStream {
                                             address: Address::from_str(&fs.address).expect(
```

Comment:
It would be better to throw an error here rather than panicking, if possible. Can we replace with `anyhow::anyhow!(...)?`?

Reply:
I think this is OK because an error here would be unrecoverable (this is in the testnet generation command). I'm also unsure how to bubble up the `Err` response to the scope outside of the `map`.